src/test/java/org/bonitasoft/connectors/scripting/ShellConnectorTest.java

```diff
@@ -184,8 +184,8 @@ private static String multiLineShellScript() {
         }
     }
 
-    private static String script(String... elements) {
-        return join(lineSeparator(), elements);
+    private static String script(String... scriptLines) {
```

Comment:
I was more thinking of renaming the method itself...

Reply:
😄

Reply:
can you propose something, that seemed clear to me 😨